src/libs/Environment/betaChecker/index.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * There's no beta build in non native
+ *
+ * @returns {Promise}
+ */
+function isBetaBuild() {
+    return new Promise(resolve => resolve(false));
```

Comment:
I think this can simply be
```suggestion
    return Promise.resolve(false);
```

Reply:
You'll have to double-check that it works, but I think it does according to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/resolve